packages/web/server/addToCRM.ts

```diff
@@ -125,10 +125,10 @@ const CUSTOM_FIELD_IDS = {
   source: 1,
 }
 async function setCustomFields(contactID: ActiveCampaignID, fields) {
-  Object.keys(fields).map((key) => {
+  Object.keys(fields).map(async (key) => {
```

Comment:
Are you sure there aren't any weird effects of using async await in an map function?

Reply:
Afaik, it should be safe, I added `await Promise.all` to wait for all these calls to finish.

Reference: https://flaviocopes.com/javascript-async-await-array-map/